Documentation/devel/release.md

```diff
@@ -19,12 +19,14 @@ Let's get started:
 
 The rkt version is [hardcoded in the repository](https://github.com/coreos/rkt/blob/master/configure.ac#L2), so the first thing to do is bump it:
 
-- Run `scripts/bump-release v0.16.0`.
-  This should generate two commits: a bump to the actual release (e.g. v0.16.0), and then a bump to the release+git (e.g. v0.16.0+git).
+- Run `scripts/bump-release v1.1.0`.
+  This should generate two commits: a bump to the actual release (e.g. v1.1.0), and then a bump to the release+git (e.g. v1.1.0+git).
   The actual release version should only exist in a single commit!
 - Sanity check what the script did with `git diff HEAD^^` or similar.
   As well as changing the actual version, it also attempts to fix a bunch of references in the documentation etc.
-- Fix the commit `HEAD^^` so that the version in `stage1/aci/aci-manifest.in` is correct.
+- Fix the commit `HEAD^` so that the version in `stage1/aci/aci-manifest.in` is the version of appc/spec vendored with rkt.
+- Fix the commit `HEAD^` so that the version in `scripts/acbuild-rkt-builder.sh` and `scripts/build-rir.sh` is correct (`1.1.0`).
```

Comment:
It works, but needs a `configure.ac`-like handling, we want `v1.1.0` in the first commit and `v1.1.0+git` in the second one. Right now it just does the dumb `s/1.0.0/1.1.0`.